### PR TITLE
Emails WIP

### PR DESCRIPTION
**Pull request by screaser**
OK, I think I have the two primary recurring emails ready to all review
together.

Both of these (morning sales report, bi-weekly inventory report) are
fully dynamic in their templates; currently operating off data in
/lib/app-server-globals.js.

Obviously I need to pull from the DB; but that’s not quite ready… in
the meantime let’s try to finalize the content and design of these
things, for version 1.0.

### CODE REVIEW

**Comment by screaser** [comment]
OK to merge this one? It's a WIP but it should be stable...

File: services/mailer/server/app.js [hunk 1]
```diff
@@ -78,4 +77,6 @@ SyncedCron.add({
   }
 });
 
-SyncedCron.start();
\ No newline at end of file
+SyncedCron.start();
+
+myData = new Mongo.Collection('test_table');
```
Comment (bzwheeler) [inline services/mailer/server/app.js]: What is `test_table` being used for?

File: services/mailer/server/lib/app-server-globals.js [hunk 2]
```diff
@@ -1,51 +1,191 @@
...
       { 
         name: 'PNP Sheet',
         img: 'absolute.url.to.product.image',   // TEMP SCR
         sales: 13,
-        inStock: 0,
-        inbound: 100
+        returns: 1,
+        profit: 84,
+        promoCost: 0,
+        bars: "bars-good"
```
Comment (bzwheeler) [inline services/mailer/server/lib/app-server-globals.js]: Are these "bars" things temporary placeholders until we can build "real" charts?

File: services/mailer/server/app.js (hunk 1 above)
Comment (screaser) [inline services/mailer/server/app.js]: Hmm; that appears to be an accidental inclusion. Sorry -- good catch.

File: services/mailer/server/lib/app-server-globals.js (hunk 2 above)
Comment (screaser) [inline services/mailer/server/lib/app-server-globals.js]: It's one concept -- I wanted to look over those with you together and decide whether this direction is good, or if we need a different approach... they're just supposed to give you a quick sense of "direction/momentum" (but they may be too hi-res for that).

Comment (bzwheeler) [inline services/mailer/server/lib/app-server-globals.js]: Ok cool, yeah, maybe too hi-res for that purpose.  Going to merge but definitely worth catching up soon on "all the things" :grinning:

**Comment by nathangreenling** [comment]
You guys are awesome :).

Nate

**Pull request merged**